docs/howto/krustlet-on-microk8s.md

```diff
@@ -9,6 +9,20 @@ MicroK8s and the Krustlet, on a single machine. `kubectl` is required but is ins
 as `microk8s.kubectl`. The following instructions use `microk8s.kubectl` for simplicity.
 You may use a standlone `kubectl` if you prefer.
 
+In order for the bootstrap authentication token to work, your kube-apiserver needs to have
+the `--enable-bootstrap-token-auth`
+(see [bootstrap-tokens](https://kubernetes.io/docs/reference/access-authn-authz/bootstrap-tokens/)).
+To verify you have it enabled, check the process args:
+```bash
+$ ps -ef | grep kube-apiserver | grep "enable-bootstrap-token-auth"
+```
+If it doesn't show up and you installed using `snap` you can find the startup args in
+`/var/snap/microk8s/current/args/kube-apiserver` and add the flag. Now you need to
```

Comment:
Can you please start the new sentence in another paragraph? The first sentence talks about finding the startup arguments, and this sentence discusses a separate topic (restarting kube-apiserver), so it should be a separate paragraph.